### ISSUE TITLE
Code formatting profile

### ISSUE THREAD

**Issue by msavy** [apiman/apiman-cli#26]
I noticed you guys have a particular style for laying out things such as lambdas, streams, etc. 

Do you have a code formatting profile you could share? For example, eclipse, IntelliJ, etc.

**Comment by outofcoffee** [comment]
Yes, we should share the IntelliJ formatting config we're using.
